detox/test/e2e/30.custom-keyboard.test.js

```diff
@@ -3,12 +3,13 @@ describe(':ios: Custom Keyboard', () => {
     await device.reloadReactNative();
     await element(by.text('Custom Keyboard')).tap();
   });
-  
+
   afterEach(async () => {
     await element(by.id('closeButton')).tap();
   });
 
-  it('should interact with keyboard when field is first responder', async () => {
+  // TODO: fix this test
+  it.skip('should interact with keyboard when field is first responder', async () => {
```

Comment:
@asafkorem, unfortunately, something is broken here:

```js
await element(by.id('keyboardHelloButton')).tap();
```

Could you help me, please?

Reply:
you can revert this skip now :)